auth/providers/azure/graph/msi_tokenprovider.go

```diff
@@ -96,9 +96,15 @@ func (u *msiTokenProvider) Acquire(ctx context.Context, token string) (AuthRespo
 	}
 
 	authResp.TokenType = tokenResp.TokenType
-	authResp.Expires, err = strconv.Atoi(tokenResp.ExpiresIn)
+	// This value is true only at token creation time, if a cached token is used, its not accurate
+	authResp.ExpiresIn, err = strconv.Atoi(tokenResp.ExpiresIn)
```

Comment:
IMO if we don't use this, we should remove it from the parsed responsed json payload to avoid using it elsewhere